hpx/runtime/threads/policies/scheduler_base.hpp

```diff
@@ -261,7 +279,7 @@ namespace hpx { namespace threads { namespace policies {
         // the scheduler mode is simply replicated across the cores to
         // avoid false sharing, we ignore benign data races related to this
         // variable
-        std::vector<util::cache_line_data<std::atomic<scheduler_mode>>> modes_;
+        std::atomic<scheduler_mode> mode_;
```

Comment:
It might still be good to leave this on its own cache line to keep it separate from the mutexes below.

Reply:
And the comment above needs to be updated...

Reply:
good catch. I meant to have it in a cache line thingy, but completely forgot.